api/service/projects_service_test.go

```diff
@@ -103,7 +111,15 @@ func TestProjectsService_CreateProject(t *testing.T) {
 			storage.On("Save", tt.expResult).Return(tt.expResult, nil)
 
 			authEnforcer := &enforcerMock.Enforcer{}
-			projectsService, err := NewProjectsService(MLFlowTrackingURL, storage, authEnforcer, tt.authEnabled)
+
+			projectsService, err := NewProjectsService(
+				MLFlowTrackingURL, storage, authEnforcer, tt.authEnabled,
+				config.UpdateProjectConfig{
+					Endpoint:         "",
+					PayloadTemplate:  "",
+					ResponseTemplate: "",
+				},
+			)
```

Comment:
If all the values are going to be the empty string, you don't actually have to initialise them explicitly.